packages/server/lib/config_options.ts

```diff
@@ -74,11 +74,6 @@ export const options = [
     defaultValue: false,
     validation: v.isBoolean,
     isExperimental: true,
-  }, {
-    name: 'experimentalRunEvents',
-    defaultValue: false,
-    validation: v.isBoolean,
-    isExperimental: true,
```

Comment:
would it be good to add this to `breakingOptions` at the bottom of this file, so users get a warning that it's no longer required to pass this option?